### ISSUE TITLE
parse .vmdl_c instead of .vphys

### ISSUE THREAD

**Issue by salimhankurul** [AtomicBool/cs2-map-parser#4]
.vphys_c files are deprecated.
new data is on .vmdl_c files, i was wondering would you consider making this change on your repo ?

**Comment by Xsintashi** [comment]
Community maps seems to be using vmdl. Ancient was updated on 12.07 and still has vphys

**Comment by salimhankurul** [comment]
Yes, They still have vphys but its outdated, and no longer receives changes. all maps now keep their newest data on .vmdl_c

**Comment by atombottle** [comment]
One way is open it in source2viewer and copy from the "PHYS" tab

**Comment by Xsintashi** [comment]
> Yes, They still have vphys but its outdated, and no longer receives changes. all maps now keep their newest data on .vmdl_c

vphys still work, its problem from your side

**Comment by salimhankurul** [comment]
@atombottle you misunderstood, i know how to decompile .vmdl_c files using source2viewer.
That was not the issue we were talking about.
Read the thread again please :D 


.vphys are outdated for all maps,
not just community maps.
we should use .vmdl_c

**Comment by atombottle** [comment]
> @atombottle you misunderstood, i know how to decompile .vmdl_c files using source2viewer. That was not the issue we were talking about. Read the thread again please :D
> 
> .vphys are outdated for all maps, they are no longer being updated with latest changes. not just community maps. we should use .vmdl_c

![image](https://github.com/user-attachments/assets/c5288026-2091-427e-8522-7dcd67b5d0fa)
I mean source2viewer have decompiler for vmdl_c to phys

**Comment by atombottle** [comment]
even if I just export vmdl_c, source2viewer still gives phys 

> @atombottle you misunderstood, i know how to decompile .vmdl_c files using source2viewer. That was not the issue we were talking about. Read the thread again please :D
> 
> .vphys are outdated for all maps, they are no longer being updated with latest changes. not just community maps. we should use .vmdl_c

You're right but what I'm tring to say is I have no idea of dmx parser but a temp way to fix it by using source2viewer, it will convert vmdl_c to kv3 style like the vphys. it will take some time but works for me on dust2 and new map. even if I just export it without decomplie it with the "vphys" in vpk.

**Comment by salimhankurul** [comment]
Yea i understand you now alright, alright
I copied data on PHYS in .vmdl_c, pasted it to .txt file, saved that txt file as "world_physics.vphys" used your parser to create .tri file, it worked d

**Comment by exp1007** [comment]
Exporting the PHYS much easier by editing source2viewer source. Its exported to documents
Add in Resource.cs line 433:
```c++
            string filePath = Environment.GetFolderPath(Environment.SpecialFolder.MyDocuments);

            if(block.Type == BlockType.PHYS)
            {
                File.WriteAllText(filePath + "\\source2viewer\\world_physics.vphys", block.ToString());
            }
 ```

**Comment by JanEricNitschke** [comment]
@exp1007 Where is that place in the code at the moment? And would you be able to make a PR to the Source2Viewer repo?
